nexus/src/app/sagas/mod.rs

```diff
@@ -400,3 +401,18 @@ where
     )
     .await
 }
+
+/// Reliable persistent workflows can request that sagas be run as part of their
+/// activation by sending a SagaRequest through a supplied channel to Nexus.
+pub enum SagaRequest {
+    #[cfg(test)]
+    TestOnly,
+}
+
+impl SagaRequest {
+    pub fn channel() -> (mpsc::Sender<SagaRequest>, mpsc::Receiver<SagaRequest>)
+    {
+        // XXX this is a magic number!
```

Comment:
Could we have a simple explanation for how this was chosen?

Shoving this into a named const with a comment like "This limits _____, and was chosen arbitrarily" would be fine - at least that would identify who gets blocked on it!

Reply:
I've changed this in 5ee010555, this is a good point.